### DISCUSSION TITLE
NGINX or APACHE

### DISCUSSION THREAD

**Discussion by maglovskiNenad** [bregman-arie/devops-exercises#10226] [Ideas]
It would probably be a solid idea to if is it possible to add a section like NGINX or APACHE.

**Comment by thecoderbutcher** [comment]
In many cases, the choice between NGINX and Apache comes down to personal preference, specific use case requirements, and the expertise of your team. Some environments even use both, with one serving as the main web server and the other as a reverse proxy or for specific tasks.

**Comment by maglovskiNenad** [comment]
Exactly and you're 100% right, of course I wouldn't change anything in what you wrote except that you just gave an excellent example of how it can be used in different ways in different environments. Preference or not, there are actually some things that might still bring a little experience to someone somewhere. Of course if they are mentioned.
